src/features/movieDetails/index.js

```diff
@@ -1,77 +1,100 @@
 import { MovieDescriptionTile } from "../../common/Tiles";
 import PersonTile from "../../common/Tiles/PersonTile";
+import Loader from "../../common/Loader";
+import { Error } from "../../common/Error";
+import { nanoid } from "@reduxjs/toolkit";
 
 const MovieDetails = () => {
+    const secureBaseUrl = "https://image.tmdb.org/t/p/original";
     const dispatch = useDispatch();
     const { id } = useParams();
 
     useEffect(() => {
         dispatch(getDetailsForMovie({ movieId: id }));
     }, [id, dispatch]);
+    const status = useSelector(selectStatus);
     const details = useSelector(selectDetails);
-    //const cast = useSelector(selectCast);
-    //const crew = useSelector(selectCrew);
+    const cast = useSelector(selectCast);
+    const crew = useSelector(selectCrew);
 
     return (
-        <p>{details.original_title}</p>
-
-        /* <MainWrapper
-            backdrop={
-                <BackdropWrapper>
-                    <Title>
-                        {details.original_title}
-                    </Title>
-                    <PosterWrapper>
-                        <Backdrop
-                            src={poster}
+        status === "loading" ?
+            <Loader /> :
+            status === "error" ?
+                <Error /> :
+                <MainWrapper
+                    backdrop={
+                        <BackdropWrapper>
+                            <Title>
+                                {details.original_title}
+                            </Title>
+                            <PosterWrapper>
+                                <Backdrop
+                                    src={
+                                        details.backdrop_path ?
+                                            `${secureBaseUrl}${details.backdrop_path}` :
+                                            ""//`${noPerson}`
+                                    }
+                                    alt="backdrop"
+                                />
+                            </PosterWrapper>
+                            <BackdropVotes
+                                voteAverage={details.vote_average}
+                                voteCount={details.vote_count}
+                            />
+                        </BackdropWrapper>
+                    }
+                    content={
+                        <MovieDescriptionTile
+                            posterPath={`${secureBaseUrl}${details.poster_path}`}
+                            title={details.original_title}
+                            year={(details.release_date || "").substring(0, 4)}
+                            genres={
+                                (details.genres).map(genre => genre.name)
+                            }
+                            voteAverage={details.vote_average}
+                            voteCount={details.vote_count}
+                            description={details.overview}
+                            firstValue={"China USA"}
```

Comment:
`firstValue` should be `details.production_countries`
so in the code: 
`firstValue={(details.production_countries).map(country =>"${country.name}, ")}` ( change double quotes to ``)

Reply:
Sorry, I forgot to come back to this